dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -276,7 +276,15 @@ def find_all_providers_affected(
             "to make changes to the provider."
         )
         get_console().print(f"Suspended providers: {suspended_providers}")
-        sys.exit(1)
+        if fail_if_suspended_providers_affected:
+            get_console().print(
+                "[error]This PR did not have `allow suspended provider changes` " "label set so it will fail."
```

Comment:
```suggestion
                "[error]This PR did not have `allow suspended provider changes` label set so it will fail."
```
Some black shenanigans I think?

Reply:
Ah yeah. This is I think the only check we used to have in Pylint that we lost ..

Reply:
Maybe Ruff can do it already :thinking: